packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -213,6 +215,62 @@ class MediaQueryData {
   ///   property and how it relates to [padding] and [viewInsets].
   final EdgeInsets viewPadding;
 
+  /// The areas along the edges of the display where the system consumes
+  /// certain input events and blocks delivery of those events to the app.
+  ///
+  /// Starting with Android Q, simple swipe gestures that start within the
+  /// [systemGestureInsets] areas are used by the system for page navigation
+  /// and may not be delivered to the app. Taps and swipe gestures that begin
+  /// with a long-press are delivered to the app, but simple press-drag-release
+  /// swipe gestures which begin within the area defined by [systemGestureInsets]
+  /// may not.
+  ///
+  /// Apps should avoid locating gesture detectors within the system gesture
+  /// insets area. There is no reason to avoid putting visual elements within
```

Comment:
[my fault] "There is no reason" is probably overstating it. Maybe: Apps should feel free to put visual elements within